### PR TITLE
CMT-nek PSAAP II update

### PR DESCRIPTION
**Pull request by hackljf**
Please let me know your wishes regarding the LPM particle solver. Their author, David Zwick, has been given guidelines that I am not familiar with. If LPM needs to be removed from core/, mkuserfile and makefile.template, I will do that upon request and put them in a branch that stays with UF.

### CODE REVIEW

**Comment by stgeke** [comment]
All the LPM stuff is in this PR #567. Please remove  it from this PR.

**Comment by hackljf** [comment]
LPM removed from core in 58944ec

https://github.com/Nek5000/Nek5000/pull/571/commits/58944ec339250899d8d6d1594cdd92b1ad730ca9

File: tools/maketools [hunk 1]
```diff
@@ -1,17 +1,17 @@
 #!/bin/bash
```
Comment (stgeke) [inline tools/maketools]: Please revert the changes to this file

File: tools/genbox/SIZE [hunk 2]
```diff
@@ -1,5 +1,5 @@
-      parameter (mbox  = 10)            ! Max number of Boxes in .box file
-      parameter (maxx  = 1000)          ! Max elements in one direction
+      parameter (mbox  = 5)            ! Max number of Boxes in .box file
```
Comment (stgeke) [inline tools/genbox/SIZE]: Please revert the changes to this file

File: core/postpro.f [hunk 3]
```diff
@@ -376,7 +376,7 @@ subroutine filter_s0(scalar,wght,ncut,name5) ! filter scalar field
       call filterq(scalar,intv,lx1,lz1,wk1,wk2,intt,if3d,fmax)
       fmax = glmax(fmax,1)
 
-      if (nio.eq.0) write(6,1) istep,fmax,name5
+c     if (nio.eq.0) write(6,1) istep,fmax,name5
```
Comment (stgeke) [inline core/postpro.f]: Please revert the changes to this file

File: core/navier5.f [hunk 4]
```diff
@@ -871,8 +871,8 @@ subroutine build_new_filter(intv,zpts,nx,kut,wght,nio)
       enddo
       np1 = nx+1
       if (nio.eq.0) then
-         write(6,6) 'filt amp',(pht (k),k=1,nx*nx,np1)
-         write(6,6) 'filt trn',(diag(k),k=1,nx*nx,np1)
+c        write(6,6) 'filt amp',(pht (k),k=1,nx*nx,np1)
```
Comment (stgeke) [inline core/navier5.f]: Please revert the changes to this file

**Comment by stgeke** [comment]
All tests failed:
/home/travis/build/Nek5000/Nek5000/core/mkuserfile: line 117: syntax error: unexpected end of file

**Comment by hackljf** [comment]
Thanks for catching this. I'm sorry I couldn't find it the first times.
